### ISSUE TITLE
Angular 13 support

### ISSUE THREAD

**Issue by inexuscore** [swimlane/ngx-charts#1699]
Any idea when Angular 13 will be supported? We're trying to update to Angular 13 but a few libraries have conflicting peer dependencies. It would be great to bump the versions and target Angular 13 if not difficult. Thank you very much.

**Comment by jrobens** [comment]
Specifically  Could not resolve dependency:
npm ERR! peer rxjs@"6.x" from @swimlane/ngx-charts@19.1.0

**Comment by internalsystemerror** [comment]
Most projects are simply extending the rxjs version constraint from `6.x` to `6.x || 7.x`. Would that be possible here so that we can use RXJS 7 with our projects?

**Comment by cromefire** [comment]
It's also still using the older library format:
```
Processing legacy "View Engine" libraries:
- @swimlane/ngx-charts [es2015/esm2015] (git+https://github.com/swimlane/ngx-charts.git)
Encourage the library authors to publish an Ivy distribution.
```

**Comment by blogcraft** [comment]
Hi @marjan-georgiev is this in the works?

**Comment by michaelfaith** [comment]
> It's also still using the older library format:
> 
> ```
> Processing legacy "View Engine" libraries:
> - @swimlane/ngx-charts [es2015/esm2015] (git+https://github.com/swimlane/ngx-charts.git)
> Encourage the library authors to publish an Ivy distribution.
> ```

Opened a new issue for this, in case this open PR closes the issue without solving the bigger problem.  https://github.com/swimlane/ngx-charts/issues/1708

**Comment by inexuscore** [comment]
Has anyone tested the latest release on Angular 13? I just checked the latest commits and it seems we can now use Angular 13. If anyone has tested it this it would be great to hear it's safe and sound to upgrade to Angular 13 now. Thank you very much, I'll give it a try ASAP and share my experience.

**Comment by cromefire** [comment]
I've been using it for some time on Angular 13 now and even though it's not officially compatible it's fine, although (the old version at least) will not benefit from the build speed (and maybe build size) improvements with the new format.

**Comment by michaelfaith** [comment]
> Has anyone tested the latest release on Angular 13? I just checked the latest commits and it seems we can now use Angular 13.
No, the latest commits haven't been released yet.  There hasn't been a release since those changes were merged in last week.  That's a different question, though, than whether or not the old version can work with Angular 13.  It can because ngcc will convert it for you.  But it won't have the new APF and Angular 13 improvements until those changes are published in a release.

**Comment by inexuscore** [comment]
@michaelfaith fabulous! if the latest release works with Angular 13 that's more than enough for us at the moment. Thank you very much 🍻

**Comment by jordykrul** [comment]
Okay, as of my understanding there was a fix (#1707 ). But this fix resulted in errors which were resolved in #1717. Essentially the fix got reverted. So this means that the current version of ngx-charts still doesn't support Angular 13+. That's correct wright? When can we expect support?
